src/App.css

```diff
@@ -38,6 +37,12 @@
 .search {
   padding: 5px 0 20px 0;
 }
+.searchBtn {
+  background-color: rgb(201, 198, 203);
+  width: 120px;
+  height: 35px;
+  border-radius: 5px;
+}
```

Comment:
Typically CSS classes use pascal case, `.SearchBtn`, `.Search`, and `.AppLogoImg`